gradle-config/aiven-pmd-test-ruleset.xml

```diff
@@ -37,7 +37,7 @@
     </rule>
     <rule ref="category/java/design.xml/CognitiveComplexity">
         <properties>
-            <property name="reportLevel" value="20" />
+            <property name="reportLevel" value="21" />
```

Comment:
Test refactoring caused the complexity to grow on the test function `IntegrationTest.groupByKey`.